Makefile

```diff
@@ -1,40 +1,62 @@
 SHELL := /bin/bash
+.DEFAULT_GOAL := build
 
-PACKAGES=$(shell go list ./... | grep -v /vendor/)
-TESTARGS?=-race
-CURRENT_DIR = $(shell pwd)
-SOURCEDIR = $(CURRENT_DIR)
+PACKAGES = $(shell go list ./... | grep -v /vendor/)
+
+TESTARGS ?= -race
+
+CURRENTDIR = $(shell pwd)
+SOURCEDIR = $(CURRENTDIR)
 SOURCES := $(shell find $(SOURCEDIR) -name '*.go')
 APP_SOURCES := $(shell find $(SOURCEDIR) -name '*.go' -not -path '$(SOURCEDIR)/vendor/*')
-VERSION=$(shell cat .goxc.json | python -c "import json,sys;obj=json.load(sys.stdin);print obj['PackageVersion'];")
+
+PATH := $(CURRENTDIR)/bin:$(PATH)
```

Comment:
setting this path once for all tests, thanks to running builds with consul on travis we got +10% to coverage for free ;)